scripts/build-storybook.js

```diff
@@ -2,11 +2,41 @@ const dotenv = require("dotenv");
 const fs = require("fs");
 const { execSync } = require("child_process");
 
+const whitelistArgs = [
+  "-h",
+  "--help",
+  "-V",
+  "--version",
+  "-o",
+  "--output-dir",
+  "-c",
+  "--config-dir",
+  "--loglevel",
+  "--quiet",
+  "--debug",
+  "--debug-webpack",
+  "--stats-json",
+  "--docs",
+  "--test",
+  "--ci",
+  "--smoke-test",
+  "--preview-url",
+  "--force-build-preview",
+  "--disable-telemetry",
+  "--enable-crash-reports",
+  "--webpack-stats-json",
+];
 const envConfig = dotenv.parse(fs.readFileSync(".env"));
 process.env.STORYBOOK_BUILD = envConfig.STORYBOOK_BUILD;
+const args = process.argv.slice(2);
+const storybookArgs = args
+  .filter((arg) => whitelistArgs.includes(arg.split("=")[0]))
```

Comment:
**suggestion**: what do you think about this tweak? This would help ensure that non key-value arguments like `--ci` are not filtered out:

```suggestion
  .filter((arg) => {
      const [key] = arg.split("=");
      return whiteListArgs.includes(key) || whiteListArgs.includes(arg)
  })
```